### ISSUE TITLE
How to re-load the modified settings from the leeloolxp.com site

### ISSUE THREAD

**Issue by mudrd8mz** [anayleeloolxp/moodle-block_tb_latestentry#2]
It seems to me the only moment when settings are loaded from leeloolxp.com is when the admin saves the plugin settings. Which typically happens just once after installing the plugin.

So when something is changes - e.g. the _Interval of time considered "recent"_ value - it does not really get re-loaded to the Moodle site unless the admin can read the code and find out that they have to resave the plugin setting.

**Comment by anayleeloolxp** [comment]
Yes you are partially right over here. The plugin works as you said. 

To solve this we already have did a plugin https://moodle.org/plugins/local_leeloolxpcontentapi
This api plugin calls an api to moodle whenever settings are updated in Leeloo, telling moodle to update data from Leeloo.

So in simple words, when setting updated in Leeloo, Leeloo makes api call to local_leeloolxpcontentapi plugin, then local_leeloolxpcontentapi call the function to update settings from Leeloo and save to Moodle.

I hope I am clear and this makes sense to you.

**Comment by mudrd8mz** [comment]
Ah right. I guess the problem in my case was that my Moodle server does not have a public URL so Leeloo platform could not reach it. And there was no indication of any error.
